vpr/src/place/net_cost_handler.h

```diff
@@ -511,4 +512,17 @@ class NetCostHandler {
      */
     double get_net_wirelength_from_layer_bb_(ClusterNetId net_id);
 
+    template<typename BBT>
+    std::pair<double, double> get_chan_place_fac_(const BBT& bb) {
```

Comment:
If this has good QoR, comment this routine to explain how this works.  Also comment the template parameter (what are the types this handles).